src/test/getarg_tests.cpp

```diff
@@ -13,10 +13,17 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/test/unit_test.hpp>
 
-BOOST_FIXTURE_TEST_SUITE(getarg_tests, BasicTestingSetup)
+class LocalTestingSetup : BasicTestingSetup {
```

Comment:
I kept getting `cannot define or redeclare 'ResetArgs' here because namespace 'getarg_tests' does not enclose namespace
      'LocalTestingSetup'`

Reply:
>  I kept getting cannot define or redeclare 'ResetArgs' here because namespace 'getarg_tests' does not enclose namespace 'LocalTestingSetup'

You can put this into a namespace. Like in `validation_block_tests`